### PR TITLE
Set ingroup doc to SemanticCompoundQueries

### PR DESCRIPTION
**Pull request by PeterTheOne**

### CODE REVIEW

File: tests/phpunit/Integration/I18nJsonFileIntegrityTest.php [hunk 1]
```diff
@@ -5,7 +5,7 @@
 use SMW\Tests\Utils\UtilityFactory;
 
 /**
- * @group semantic-compound-queries
```
Comment (mwjames) [inline tests/phpunit/Integration/I18nJsonFileIntegrityTest.php]: This relates to PHPUnit https://phpunit.de/manual/4.0/en/appendixes.annotations.html#appendixes.annotations.group.

Comment (PeterTheOne) [inline tests/phpunit/Integration/I18nJsonFileIntegrityTest.php]: correct. I just wanted to change it to the same group as the other tests.

Comment (mwjames) [inline tests/phpunit/Integration/I18nJsonFileIntegrityTest.php]: @ingroup may relevant to phpdoc but it is an unknown annotation for PHPUnit.

Comment (PeterTheOne) [inline tests/phpunit/Integration/I18nJsonFileIntegrityTest.php]: Oh, haven't noticed.. `@ingroup` is used in other SemanticMedia projects also.

Comment (mwjames) [inline tests/phpunit/Integration/I18nJsonFileIntegrityTest.php]: > @ingroup is used in other SemanticMedia projects also.

Used when you generate doxygen as seen in http://doc.semantic-mediawiki.org/.

**Comment by kghbln** [comment]
> Used when you generate doxygen as seen in http://doc.semantic-mediawiki.org/.

Which reminds me of https://github.com/SemanticMediaWiki/semantic-mediawiki.org/issues/8

File: tests/phpunit/Integration/I18nJsonFileIntegrityTest.php (hunk 1 above)
Comment (PeterTheOne) [inline tests/phpunit/Integration/I18nJsonFileIntegrityTest.php]: Ok, so for PHPUnit I will set use `@group` tags..

**Comment by PeterTheOne** [comment]
@kghbln What needs to be done to get this project ready for doxygen? Or should this be discussed elsewhere?

**Comment by kghbln** [comment]
@PeterTheOne I tried to do it yesterday but it was a big fail. I added the extension to packagist, managed to set up auto sync but they do not pick up the tags so composer fails to install a stable version which I use to create the docu via doxygen.

@mwjames A way out is to generate doxygen from master. Is this acceptable or even preferable. Perhaps you could also have a peep why packagist does not pick up the tags.

**Comment by mwjames** [comment]
> A way out is to generate doxygen from master. Is this acceptable or even preferable. Perhaps you could also have a peep why packagist does not pick up the tags.

Well, I don't think you will have access to tags before the creation of the `composer.json`. How would packagist and for that matter Composer work without a `composer.json` file in the replicated repo.

See https://packagist.org/packages/mediawiki/semantic-media-wiki, everything before the `composer.json` creation is not listed as available tag.

Until you make an "official" release that includes a `composer.json`, one has only access via `dev-master`.

**Comment by kghbln** [comment]
> Until you make an "official" release that includes a composer.json, one has only access via dev-master.

@mwjames Ah, I knew that there was something in the water. Did not think of this. Thank you for the insight.

**Comment by JeroenDeDauw** [comment]
As far as I can tell there are no objections to this being merged

**Pull request merged**